server/data/healthAndMedicationRestApiClient.ts

```diff
@@ -19,9 +21,12 @@ export default class HealthAndMedicationRestApiClient implements HealthAndMedica
     if (nameAndNumber) sort = `prisonerName,${nameAndNumber.toLowerCase()}`
     if (location) sort = `location,${location.toLowerCase()}`
 
-    return this.restClient.post<PagedList<HealthAndMedicationForPrison>>({
-      path: `/prisons/${prisonId}`,
-      data: { page, size: showAll ? 99999 : size, sort },
-    })
+    return this.post<PagedList<HealthAndMedicationForPrison>>(
+      {
+        path: `/prisons/${prisonId}`,
+        data: { page, size: showAll ? 99999 : size, sort },
+      },
+      asUser(token),
```

Comment:
Don't want this to be a user token - you can pass the raw client token in instead: https://github.com/ministryofjustice/hmpps-typescript-lib/blob/main/packages/rest-client/README.md#authentication